parquet/src/main/java/org/apache/iceberg/parquet/Parquet.java

```diff
@@ -231,6 +252,39 @@ private WriteBuilder createContextFunc(
       return this;
     }
 
+    private FileEncryptionProperties createEncryptionProperties(
+        NativeFileCryptoParameters nativeParameters) {
+      Preconditions.checkArgument(nativeParameters != null, "Null native crypto parameters");
+
+      ParquetCipher parquetEncryptionAlgorithm;
+      if (nativeParameters.encryptionAlgorithm() == null) {
+        parquetEncryptionAlgorithm = ParquetCipher.AES_GCM_V1; // default
```

Comment:
Question: Do we want this default to be configurable? It is always strange to hardcode these values

Reply:
In a previous draft version, we had this configuration, but it is removed for now. Couple of reasons: 
- currently, there is no significant technical reason to use the other Parquet algorithm (GCM-CTR, it is less secure that pure GCM; and the speed is basically the same in the modern Java versions)
- one less parameter for an end user to learn and set